titans/dataloader/imagenet/dali_imagenet.py

```diff
@@ -0,0 +1,192 @@
+from nvidia.dali.pipeline import Pipeline
+from nvidia.dali.plugin.pytorch import DALIClassificationIterator, LastBatchPolicy
+import nvidia.dali.fn as fn
+import nvidia.dali.types as types
+import nvidia.dali.tfrecord as tfrec
+import torch
+import numpy as np
+from titans.dataloader.utils import RandAugment
+
+__all__ = ['DaliDataloaderWithRandAug', 'DaliDataloader']
+
+
+class DaliDataloaderWithRandAug(DALIClassificationIterator):
```

Comment:
I would recommend having a `build_dali_imagenet_dataloader` for consistency as I see similar functions are given for the torchvision datasets.